docs/search_relevance/query_design.md

```diff
@@ -24,10 +24,35 @@ Each query has:
 
   * **Stable:** The current implementation of the query meets the
 
-    expectations and has been measured in the world as doing so.   
+    expectations and has been measured in the world as doing so.
 
 ## Intentions & Expectations
 
+### Shingled titles
+
+| Status | Query name | Ranking evaluation test |
+| :--- | :--- | :--- |
+| TODO | Shingles | TBD |
+
+#### Intentions
+
+Searching for works with structurally important features in the query. For example: a query for `east london` should give higher scores to instances of the phrase with tokens in that order than works which contain the tokens without the given structure. This should be particularly useful for users looking for specific titles, where word-order matters most.
```

Comment:
Could we keep the research intentions and examples / expectations separate? 

I quite like the structure of "This is what I am trying to do, and this is what I expect when trying to do it". Or does it feel unclear when you separate it that way?

Reply:
Happy to take out the middle sentence if you'd rather keep examples separate. Just so we're clear though,

> "This is what I am trying to do, and this is what I expect when trying to do it"

Do you mean "I, the user" or "I, the query designer"?

Reply:
What is the intention  of the person searching.
And what is their expectation.

I prefer to not use I or them, and just write the intention, but that's my style rather than it being useful.

Reply:
👍